.gitignore

```diff
@@ -308,4 +309,4 @@ BenchmarkDotNet.artifacts/
 
```

Comment:
do you want to update building.md where it says "this repository does not contain a solution file"

btw, are things useable if you open this? the building.md steers towards creating sub-solutions on the fly, but perhaps VS can actually work OK with the whole thing.

Reply:
Sure I can look at updating that too.

I did open it in VS and build it to check it compiled.